src/SSW.Rewards.Domain/Entities/SubmittedQuizAnswers.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SSW.Rewards.Domain.Entities;
+public class SubmittedQuizAnswers : BaseEntity
```

Comment:
Should be singular (`SubmittedQuizAnswer`) but I would rename it to `QuizUserAnswer` or something

Reply:
Partially done - made singular and added the explicit FKs. But I didn't rename it.